astroplan/plots/sky.py

```diff
@@ -71,6 +73,10 @@ def plot_sky(target, observer, time, ax=None, style_kwargs=None,
         offset will increase in the same direction as azimuth
         (see ``north_to_east_ccw`` option).
 
+    style_sheet : dict or `None` (optional)
```

Comment:
The [matplotlib docs use the name style sheet](http://matplotlib.org/users/style_sheets.html).